openfisca_core/scenarios.py

```diff
@@ -535,7 +529,11 @@ def make_json_or_python_to_axes(tax_benefit_system):
                                     conv.not_none,
                                     ),
                                 # TODO: Check that period is valid in params.
-                                period = periods.json_or_python_to_period,
+                                period = conv.condition(
+                                    conv.test_isinstance(type(None)),
+                                    conv.noop,
+                                    conv.function(periods.period),
+                                    ),
```

Comment:
Is it really necessary? 🤔 If you put only `conv.function(periods.period)`, it will skip `None` values